spec/models/defence_request_spec.rb

```diff
@@ -165,4 +165,27 @@
       end
     end
   end
+
+  describe "retrieve requests associated with solicitor's firm" do
```

Comment:
It is preferable to maintain the 4 phase setup in tests as much as possible. Having scenarios that consist entirely of expectations obfuscates reading. In this particular case, there is not much to gain from extracting the before block, for instance.